pkg/apis/serving/v1alpha1/knativeserving_types.go

```diff
@@ -28,6 +28,15 @@ const (
 	DeploymentsAvailable apis.ConditionType = "DeploymentsAvailable"
 )
 
+// Registry defines image overrides of knative images.
+// The default value is used as a default format to override for all knative deployments.
+// The override values are specific to each knative deployment.
+// +k8s:openapi-gen=true
+type Registry struct {
+	Default  string            `json:"default,omitempty"`
```

Comment:
Can we add comments to both of these properties explaining default vs override? Being particular about this because this is what will end up in API docs (which is what users will see).

Reply:
Done.